lib/dpl/provider/now.rb

```diff
@@ -0,0 +1,136 @@
+module DPL
+    class Provider
+      class Now < Provider
+        npm_g 'now'
+  
+        def directory
+          File.expand_path( (context.env['TRAVIS_BUILD_DIR'] || '.' ) + '/' + (options[:directory] || '') )
+        end
+  
+        def deploy_name
+          options[:deploy_name]
+        end
+  
+        def team
+          if options[:team]
+            log "> Adding custom team scope #{options[:team]}"
+            now_team_option = "--team #{options[:team]}"
+          else
+            now_team_option = ''
+            log '> No custom team scope provided.'
+          end
+
+          return now_team_option
+        end
+  
+        def type
+          options[:type]
+        end
+  
+        def alias_url
+          options[:alias]
+        end
+  
+        def cleanup
+          options[:cleanup]
+        end
+  
+        def scale
+          options[:scale]
+        end
+
+        def rules_domain
+          options[:rules_domain]
+        end
+
+        def rules_file
+          options[:rules_file]
+        end
+  
+        def token
+          context.env['NOW_TOKEN']
+        end
+
+        def auth
+          return "--token #{token} #{team}"
+        end
+
+        def deploy_options
+          now_deploy_options = '--no-clipboard'
+          if deploy_name
+            log "> Adding deployment name #{deploy_name}"
+            now_deploy_options = now_deploy_options + " --name #{deploy_name}"
+          else
+            log '> No deployment name provided. The directory will be used as the name'
+          end
+
+          if type
+            log "> Adding deployment type #{type}"
+            now_deploy_options = now_deploy_options + " --#{type}"
+          else
+            log '> No deployment type provided, now.sh will try to detect it...'
+          end
+
+          return now_deploy_options
+        end
+
+        def aliasing(deployment_url)
+          if alias_url
+            log '> Assigning alias...'
+            context.shell "now alias #{auth} #{deployment_url} #{alias_url}"
+            deployment_url = "https://#{alias_url}"
+          else
+            log '> No alias provided'
+          end
+          return deployment_url
+        end
+
+        def check_auth
+          if ! token then raise Error, '> Error!! Please add NOW_TOKEN Environment Variables in Travis settings (get your token here https://zeit.co/account/tokens)' end
+        end
+  
+        def check_app
+          if ! File.directory?(directory) then raise Error, 'Please set a valid project folder path in .travis.yml under deploy: directory: myPath' end
```

Comment:
This one liner seems a bit too wide. Please break it up into shorter lines. Also, I personally prefer `unless` here to `if !`.